specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -6508,6 +6508,13 @@
           },
           "description": "Specifies an array of references to backend address pools of application gateways. A scale set can reference backend address pools of multiple application gateways. Multiple scale sets cannot use the same application gateway."
         },
+        "applicationSecurityGroups": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
+          },
+          "description": "Specifies an array of references to application security group."
```

Comment:
Is "application security group" definition the same as `SubResource`?

Reply:
yes they are same, it's a reference using id. This is consistent with other references (e.g. `applicationGatewayBackendAddressPools`, `loadBalancerBackendAddressPools `) in the same model